packages/fleet/lib/supervisor/supervisor.ts

```diff
@@ -182,15 +182,21 @@ export class Supervisor {
                     if (!configOverride) {
                         return false;
                     }
-                    // image override might have a commitId
-                    // so we need to check if the image override with commitId is the same as the node's image
-                    // or if the image override (without commitId) + current deployment commitId is the same as the node's image
-                    return !(
-                        (configOverride.image === node.image || `${configOverride.image}:${deployment.commitId}` === node.image) &&
-                        node.cpuMilli === configOverride.cpuMilli &&
-                        node.memoryMb === configOverride.memoryMb &&
-                        node.storageMb === configOverride.storageMb
-                    );
+                    // TODO: remove once values in node_config_overrides are updated to NULL in the db
+                    const imageOverride = configOverride.image?.includes(':') ? configOverride.image : undefined;
```

Comment:
the TODO only applies to this line since at the moment the default value for image is `nangohq/nango-runner` without the commitHash